frontend/src/components/BetaAlert.tsx

```diff
@@ -1,18 +1,28 @@
 import { useTranslations } from "next-intl";
+import { GridContainer, Alert as USWDSAlert } from "@trussworks/react-uswds";
 
-import FullWidthAlert from "./FullWidthAlert";
+type Props = {
+  containerClasses?: string;
```

Comment:
Added this so that each instance of the banner can be spaced appropriately on its page with utility classes for e.g. margin